Specs/Scene/ModelExperimental/PntsLoaderSpec.js

```diff
@@ -13,6 +14,7 @@ import {
   VertexAttributeSemantic,
 } from "../../../Source/Cesium.js";
 import createScene from "../../createScene.js";
+import pollToPromise from "../../pollToPromise.js";
```

Comment:
Is there a reason why this comment is in `PntsLoaderSpec`?

for the tile transform tests, we only wanted to have one set of them since it works the same for each content type. For this one I kept the `b3dm` ones since I was having difficulty getting the camera setup right for the glTF ones.

Reply:
> Is there a reason why this comment is in `PntsLoaderSpec`?

Because most of the tilesets tested in `Point3DTileContentSpec` were copied over to `PntsLoaderSpec`, except for the one with the tile transform. So if we don't duplicate it, the tileset associated with it will never get tested

Reply:
Ah I understand now. I added a test in `PntsLoaderSpec`, though it seems of limited use, since the transform is applied in the `tileset.json`, not the `.pnts` file itself.

Reply:
@ptrgags ah I see... let's just leave it there for now so we don't hold up the PR, we can move things around later.